.github/workflows/job_jax_models_tests.yml

```diff
@@ -0,0 +1,138 @@
+name: JAX/Flax Models tests
+
+on:
+  workflow_call:
+    inputs:
+      runner:
+        description: 'Machine on which the tests would run'
+        type: string
+        required: true
+      container:
+        description: 'JSON to be converted to the value of the "container" configuration for the job'
+        type: string
+        required: false
+        default: '{"image": null}'
+      model_scope:
+        description: 'Scope of models for testing.'
+        type: string
+        required: true
+
+permissions: read-all
+
+jobs:
+  JAX_Models_Tests:
+    name: JAX/Flax Models tests
+    timeout-minutes: ${{ inputs.model_scope == 'precommit' && 35 || 35 }}
+    runs-on: ${{ inputs.runner }}
+    container: ${{ fromJSON(inputs.container) }}
+    defaults:
+      run:
+        shell: bash
+    env:
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
+      OPENVINO_REPO: ${{ github.workspace }}/openvino
+      INSTALL_DIR: ${{ github.workspace }}/install
+      INSTALL_TEST_DIR: ${{ github.workspace }}/install/tests
+      MODEL_HUB_TESTS_INSTALL_DIR: ${{ github.workspace }}/install/tests/model_hub_tests
+    steps:
+      - name: Check sudo
+        if: ${{ runner.os == 'Linux' }}
+        run: if [ "$(id -u)" -eq 0 ]; then apt update && apt --assume-yes install sudo; fi
+
+      - name: Set apt retries
+        if: runner.os == 'Linux'
+        run: |
+          if [ "$(id -u)" -eq 0 ]; then
+            echo 'Acquire::Retries "10";' > /etc/apt/apt.conf.d/80-retries
+          else
+            sudo sh -c "echo 'Acquire::Retries \"10\";' >> /etc/apt/apt.conf.d/80-retries"
+          fi
```

Comment:
@akashchi 
should we move such configuration steps to test docker images?

Reply:
propose to do this in separate PR by GHA team since it relates to all FEs

Reply:
Since you are using [a self-hosted runner with Docker](https://github.com/openvinotoolkit/openvino/pull/26542/files#diff-d0d91d9d5d834d7da44b2e985aa3a6a7c6bde219e556a4284d6d77394ced5c35R386), these apt retries are already set in the Dockerfile for U22_x64: https://github.com/openvinotoolkit/openvino/blob/e611edea6f9df3088523e0c9e6a5bae3de80aa87/.github/dockerfiles/ov_build/ubuntu_22_04_x64/Dockerfile#L7 and could be removed from here.